src/main/java/vg/civcraft/mc/civmodcore/api/ItemAPI.java

```diff
@@ -57,11 +57,17 @@ public static boolean isValidItem(@Nullable ItemStack item) {
 	 * @see ItemStack#equals(Object)
 	 * */
 	public static boolean areItemsEqual(@Nullable ItemStack former, @Nullable ItemStack latter) {
```

Comment:
Is it intended that this returns false if both items are null?

Reply:
Yeah, it says so in the javadoc